### ISSUE TITLE
could only be executed from current directory

### ISSUE THREAD

**Issue by zhengwx11** [hcs64/ww2ogg#7]
when ww2ogg is executed from other path, it could not find the right codebook file.
Maybe in the code something like `char* filename = './xxx.bin'` is used, which, I suppose,  is not general enough.

Note: 
The path to the program itself is contained in the `argv[0]`. It could be useful.

**Comment by hcs64** [comment]
You're right that it only uses a relative path of "packed_codebooks.bin" which will only work if that's in the working directory.

You can work around this by giving a path with the --pcb option, the readme mentions this.
